File: app/src/main/java/com/rtukpe/fixtures/ui/competitions/CompetitionsMvpView.java
package com.rtukpe.fixtures.ui.competitions;

import com.rtukpe.fixtures.data.model.Competition;
import com.rtukpe.fixtures.ui.base.MvpView;

import java.util.ArrayList;

public interface CompetitionsMvpView extends MvpView {

    void updateCompetitions(ArrayList<Competition> competitions);

}

File: app/src/main/java/com/rtukpe/fixtures/ui/competitions/CompetitionsMvpView.kt
package com.rtukpe.fixtures.ui.competitions

import com.rtukpe.fixtures.data.model.Competition
import com.rtukpe.fixtures.ui.base.MvpView
import java.util.*

interface CompetitionsMvpView : MvpView {

    fun updateCompetitions(competitions: ArrayList<Competition>)

}
